src/api/client.ts

```
@@ -208,7 +208,8 @@ export class N8nApiClient {
       delete workflowToUpdate.id; // Remove id property as it's read-only
       delete workflowToUpdate.createdAt; // Remove createdAt property as it's read-only
       delete workflowToUpdate.updatedAt; // Remove updatedAt property as it's read-only
-      delete workflowToUpdate.tags; // Remove tags property as it's read-only
+      delete workflowToUpdate.active; // Remove active property as it's read-only
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)